Blockzilla/BrowserViewController.swift

```diff
@@ -800,7 +800,7 @@ extension BrowserViewController: URLBarDelegate {
         }
     }
 
-    func urlBarDidPressScrollTop(_: URLBar, tap: UITapGestureRecognizer) {
```

Comment:
If we do not need `urlBarDidPressScrollTop()` anymore, then that function should also be removed. However, please read below.